src/utils/string/StringUtils.js

```diff
@@ -399,6 +398,10 @@ const onOpenLink = (url) => {
   });
 };
 
+const isValidUrl = (url) => {
+  return urlRegex.test(url);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `isValidUrl` function is a good addition for URL validation. However, as mentioned earlier, consider using a well-tested library like `validator.js` for URL validation.
```diff
- const isValidUrl = (url) => {
-   return urlRegex.test(url);
- };
+ import validator from 'validator';
+ 
+ const isValidUrl = (url) => {
+   return validator.isURL(url);
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->